README.md

```diff
@@ -35,21 +39,22 @@ $ cd /Applications/Python\ [VERSION]\
 $ ./Install\ Certificates.command
 ```
 
-#### Autoconf and Binutils
+#### Autoconf
 
 Open a terminal:
 ```
 $ brew install autoconf
-$ brew install binutils
```

Comment:
@pminearo I think we should remove this, as it caused quite some problems and it isn't needed as far as I can tell.